app/src/main/res/values/strings.xml

```diff
@@ -1,10 +1,10 @@
-<resources>
+<resources>                                                         <!-- Translators, pay attention to this notes! --> 
```

Comment:
pay attention to this notes

hows about 

pay attention to these notes

Reply:
You're right! Thanks for marking that!

Reply:
your welcome

Reply:
And Everything Else Looks Good.